src/main/resources/templates/layouts/main.html

```diff
@@ -20,6 +20,9 @@
     <a href="" data-th-href="@{/}" class="page-navbar__item page-navbar__item-selected">
         <ion-icon name="home"></ion-icon>
     </a>
+    <a href="" data-th-href="@{/results}" class="page-navbar__item page-navbar__item-selected">
```

Comment:
the `page-navbar__item-selected` should only be applied when the user is on that specific page. 
`home` button has it because no other page was available.

Reply:
After 2hrs I was able to do it (I think), thanks for the correction!